### PR TITLE
[IMPROVE] Rename priorities to SLA's

### PR DESCRIPTION
**Pull request by aleksandernsilva**
## Proposed changes (including videos or screenshots)

This pull request renames the current priority feature to SLA.

## Further comments

[OC-283]

[OC-283]: https://rocketchat.atlassian.net/browse/OC-283?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/meteor/ee/client/omnichannel/slaPolicies/SlasRoute.tsx [hunk 1]
```diff
@@ -29,39 +36,41 @@ const useQuery = ({ text, itemsPerPage, current }, [column, direction]) =>
...
-	const query = useQuery(debouncedParams, debouncedSort);
-	const prioritiesRoute = useRoute('omnichannel-priorities');
+	const query = useEndpointQuery(debouncedParams, debouncedSort);
+	const SlasRoute = useRoute('omnichannel-sla-policies');
 	const context = useRouteParameter('context');
-	const id = useRouteParameter('id');
+	const id = useRouteParameter('id') || '';
 
-	const onHeaderClick = useMutableCallback((id) => {
+	const onHeaderClick = useMutableCallback((id: string) => {
```
Comment (tiagoevanp) [inline apps/meteor/ee/client/omnichannel/slaPolicies/SlasRoute.tsx:53]: Why this was not removed, if is not used?

**Review by tiagoevanp — CHANGES_REQUESTED** [review]

File: apps/meteor/ee/client/omnichannel/slaPolicies/SlasRoute.tsx (hunk 1 above)
Comment (aleksandernsilva) [reply apps/meteor/ee/client/omnichannel/slaPolicies/SlasRoute.tsx:53]: @tiagoevanp I must've missed when removing the unused code.

Comment (aleksandernsilva) [reply apps/meteor/ee/client/omnichannel/slaPolicies/SlasRoute.tsx:53]: @tiagoevanp I double checked, It is being used. If you are using Github as base, remember that unchanged code remains hidden.

Comment (tiagoevanp) [reply apps/meteor/ee/client/omnichannel/slaPolicies/SlasRoute.tsx:53]: Yes, I think something just escaped from my eyes... 😅

**Review by tiagoevanp — APPROVED** [review]

**Pull request merged**